File: LeetCode/26.删除有序数组中的重复项/26.删除有序数组中的重复项.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int n = nums.size();
        if(n == 0){
            return 0;
        }
        int now = 0;
        for(int i = 1; i < n ; i++){
            if(nums[i] == nums[now]){
                continue;
            }else{
                nums[now + 1] = nums[i];
                now++;
            }
        }
        return now + 1;
    }
};

File: LeetCode/26.删除有序数组中的重复项/26.删除有序数组中的重复项.go
func removeDuplicates(nums []int) int {
    n := len(nums)
    if n == 0 {
        return 0
    }
    now := 0
    for i:=0; i < n; i++{
        if nums[now] != nums[i] {
            nums[now + 1] = nums[i]
            now++
        }
    }
    return now + 1
}